client/state/data-layer/wpcom/posts/revisions/index.js

```diff
@@ -87,9 +87,10 @@ export const receiveSuccess = ( { dispatch }, { siteId, postId }, next, revision
  * @param {Object} action Redux action
  */
 export const fetchPostRevisions = ( { dispatch }, action ) => {
-	const { siteId, postId } = action;
+	const { siteId, postId, postType } = action;
+	const resourceName = postType === 'page' ? 'pages' : 'posts';
```

Comment:
Even if we don't support custom post types quite yet, is there a way we could express this to be less tied to posts and pages? If we hypothetically had access to `rest_base`, how would we want this data handler and the associated action creator to be written?